src/taskForm.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./taskForm.css";
+
+function taskForm() {
+  return (
+    <form className="taskForm">
+      <input
+        className="task-input"
+        type="text"
+        placeholder="Add new task "

```

Comment:
You have a space at the end of the placeholder, I mean, you have:
"Add new task "
instead of
"Add new task"